tendenci/apps/events/forms.py

```diff
@@ -1130,7 +1130,7 @@ def clean_photo_upload(self):
                 raise forms.ValidationError(_('The photo must be of jpg, gif, or png image type.'))
 
             # check the image header
-            image_type = '.%s' % imghdr.what('', photo_upload.read())
+            image_type = '.%s' % what_wagic('', photo_upload.read())
```

Comment:
Typo. Also the "import magic" at the top should be "from tendenci.apps.base.utils import what_magic".